pyanaconda/packaging/yumpayload.py

```diff
@@ -1296,6 +1296,9 @@ def _applyYumSelections(self):
         else:
             self._selectYumGroup("core")
 
+        # Select the base group by default. It might be excluded later.
+        self._selectYumGroup("base")
```

Comment:
According to the [documentation](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/sect-kickstart-syntax.html#sect-kickstart-packages), this is the correct behaviour. Due to the removed `--nobase` option, the minimal installation should look like this:
```
%packages --nocore
-@base
%end
```

I can deselect the base group for `--nocore` option, but then we should change the documentation.